packages/syntax-core/src/Checkbox/Checkbox.tsx

```diff
@@ -119,14 +119,22 @@ const Checkbox = ({
       )}
     >
       <div className={checked ? checkedStyling : uncheckedStyling}>
-        {checked && (
+        {checked && themeName === "classic" && (
           <svg aria-hidden="true" viewBox="0 0 24 24" width={iconWidth[size]}>
             <path
-              fill={themeName === "classic" ? "#fff" : "#050500"}
+              fill="#fff"
               d="m9 16.2-3.5-3.5a.9839.9839 0 0 0-1.4 0c-.39.39-.39 1.01 0 1.4l4.19 4.19c.39.39 1.02.39 1.41 0L20.3 7.7c.39-.39.39-1.01 0-1.4a.9839.9839 0 0 0-1.4 0L9 16.2z"
             ></path>
           </svg>
         )}
+        {checked && themeName === "cambio" && (
+          <svg aria-hidden="true" viewBox="0 0 24 24" width={iconWidth[size]}>
+            <path
+              fill={error ? "#6d0002" : "#050500"}
```

Comment:
should these hex values be the color vars?

Reply:
@somethiiing thanks for spotting - updated

```tsx
          <svg aria-hidden="true" viewBox="0 0 24 24" width={iconWidth[size]}>
            <path
              className={error ? styles.checkmarkError : styles.checkmark}
```